# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/scons/no-setuptools.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- x/setup.cfg.vanilla	2021-02-09 16:44:31.592024566 +0100
+++ x/setup.cfg	2021-02-09 16:44:41.799896235 +0100
@@ -36,9 +36,6 @@
 [options]
 zip_safe = False
 python_requires = >=3.5
-install_requires = 
-	setuptools
-setup_requires = setuptools
 include_package_data = True
 packages = find:
 
